product-service/src/functions/index.ts

```diff
@@ -1 +1,2 @@
-export { default as hello } from './hello';
+export { default as getProductsList } from './getProductsList';
+export { default as getProductsById } from './getProductsById';
```

Comment:
not seeing createProduct here, but it works with postman so its fine to skip export here.